### ISSUE TITLE
Delete Accessibility Standards page

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#49312]
## User Story
As a VFS team member, I want to be able to rely on the accuracy of the accessibility standards from Platform so that I can be sure my team is meeting those standards for our product.

Assignee: @briandeconinck 
Peer Reviewer: @allison0034 

## Description
Unbeknownst to the Governance team, the [Accessibility Standards](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-standards) page was created and published under the Developer docs > QA and accessibility testing > Accessibility testing section of Platform Website. This page is just a duplicate of the [VA.gov Experience Standards](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/va-gov-experience-standards) page, however it has not been maintained or updated, so it may be out of date. All Accessibility standards are published as part of the VA.gov Experience Standards and should remain there.

## Impacted Artifacts
[Accessibility Standards](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-standards)

## Tasks
- [x] Delete the Accessibility Standards page

## Peer Review
_To be completed by peer reviewer_
- [x] User story and acceptance criteria are met

## Acceptance Criteria
- [x] The Accessibility Standards page under Developer Docs has been deleted

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization), if applicable
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [x] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [x] Ticket has been moved to the Planned pipeline in the Governance team board
- [x] Ticket as an assignee and peer reviewer

**Comment by briandeconinck** [comment]
Page has been archived in Confluence. Waiting on changes to publish to Platform website.

**Comment by briandeconinck** [comment]
@allison0034 Platform website changes published, this is ready for you to review/verify that it happened.

**Comment by allison0034** [comment]
@briandeconinck I am getting a 404 when I click the artifacts link. Is that correct?

**Comment by briandeconinck** [comment]
Yep -- page was archived in Confluence and removed from Platform website.
